src/vxfw/Border.zig

```diff
@@ -5,10 +5,20 @@ const Allocator = std.mem.Allocator;
 
 const vxfw = @import("vxfw.zig");
 
+pub const BorderLabel = struct { text: []const u8, alignment: enum {
+    top_left,
+    top_center,
+    top_right,
+    bottom_left,
+    bottom_center,
+    bottom_right,
+} };
```

Comment:
Since have the other change let's add this one too: just add a comma after the enum closing brace to get zig fmt to format it more nicely.